README.md

```diff
@@ -1,19 +1,35 @@
-# core
+![Logo!](https://avatars.githubusercontent.com/u/98192226?s=200&v=4)
+
+# Authsio
+
+## Core
 
 Core authsio web application for identity & access management
 
+---
+
+### Project Plan
+
+An open source identity & access management solution that will serve as a multi tenant drop in access management solution. There are many other access management tools on the market but nothing that is multi tenant by design and can be self hosted. Today with privacy and security concerns of your users in mind you need an IAM that separates your concerns. With that in mind, our current project plan is to get a MVP out the door in the next couple months.
+
+For more information about this project click here [link](https://www.authsio.com/)
+
+---
+
 ### Deployment (WIP)
 
 We are using [Turborepo](https://turborepo.org/) to compile and build our application.
 
+---
+
 ### Developer Setup (WIP)
 
-#### Notes:
+>#### Notes
 
 - As this project is currently not feature stable, no postgres tables are currently made when running the backend application
 - Each section of the application has its own `package.json` and `tsconfig.json`
 
-#### Backend
+>#### Backend
```

Comment:
Im not sure we want to keep the block quotes here.

Reply:
was such an idea but if you do not like it, I put it out again